src/components/checkbox/Checkbox.vue

```diff
@@ -24,6 +24,7 @@
             value: {},
             nativeValue: {},
             disabled: Boolean,
+            nativeId: String,
```

Comment:
Please, remove it

Reply:
Apologies for missing that and thanks for all of your work on this excellent project!!

Reply:
Thanks!